### PR TITLE
[issue-148] - Custom 404 page

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR add new Custom 404 page.

Task: https://github.com/input-output-hk/cardano-explorer-app/issues/148

## Screenshots
Screenshot 2020-01-25 17 41 19

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1579970680064500)

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by a-rukin** [comment]
does netlify has this page?

**Comment by DmitriiGaico** [comment]
@DeeJayElly Hello!
Could you please create the pages for all the Explorer PR's so we can review and test it out?
Thank you in advance.

**Comment by DeeJayElly** [comment]
@a-rukin @DmitriiGaico will add you guys this page to storybook so you could recheck it.

File: source/styles/global/_fonts.scss [hunk 1]
```diff
@@ -5,3 +5,9 @@ $url: '/assets/fonts/Proxima-Nova-';
 @include font-face('ProximaNova', '#{$url}Semibold', 600);
 @include font-face('ProximaNova', '#{$url}Regular', normal);
 @include font-face('ProximaNova', '#{$url}Thin', 100);
+
+@include font-face(
+  'CharletComprime',
+  '/assets/fonts/ChaletComprime-CologneEighty',
+  normal
+);
```
Comment (DominikGuzei) [inline source/styles/global/_fonts.scss]: Since this font is only used on the error page it would be good to only include it there 🙏

Comment (DeeJayElly) [reply source/styles/global/_fonts.scss]: Fixed

**Comment by DeeJayElly** [comment]
> @DeeJayElly Hello!
> Could you please create the pages for all the Explorer PR's so we can review and test it out?
> Thank you in advance.

Here it is https://deploy-preview-180--cardano-explorer.netlify.com/

**Comment by DeeJayElly** [comment]
> does netlify have this page?
> 
> for me light font seems too light 🤔

Here it is https://deploy-preview-180--cardano-explorer.netlify.com/

**Comment by DmitriiGaico** [comment]
@DeeJayElly Hey! Could you please advice if this symbol is expected?
image

**Comment by DmitriiGaico** [comment]
@DeeJayElly Also comparing with Zeplin the page on Story Books looks not identical. Can you double check if it was exported correctly? Thank you.

**Comment by DeeJayElly** [comment]
> @DeeJayElly Hey! Could you please advice if this symbol is expected?
> image

Yes, design has it -> https://app.zeplin.io/project/5878c3481bb7631c20cb161a/screen/5d63ccfa82415f9b2a661f26

**Comment by DmitriiGaico** [comment]
@DeeJayElly Should we expect "Cardano Explorer" in Story book?
image

**Comment by DeeJayElly** [comment]
> @DeeJayElly Should we expect "Cardano Explorer" in Story book?
> image
> 
> Zeplin
> image

Added

**Comment by a-rukin** [comment]
image
strange BG overlap

**Comment by a-rukin** [comment]
image
@DeeJayElly can you pls explain me here how we use the fonts?
do we use two types like "ProximaNova-Light" and "ProximaNova-Semibold"? or we just tune the font weights? 
As I can see it's a bit different results when using font weight 🤔

**Comment by rhyslbw** [comment]
As this PR was merged preemptively, let's use the new issues I've created to treat changes as fixes.

**Pull request merged**